examples/bridge-app/asr/subdevice/SubDeviceManager.cpp

```diff
@@ -156,8 +156,8 @@ Protocols::InteractionModel::Status HandleWriteOnOffAttribute(SubDevice * dev, c
 {
     ChipLogProgress(DeviceLayer, "HandleWriteOnOffAttribute: attrId=%" PRIu32, attributeId);
 
-    ReturnErrorCodeIf((attributeId != OnOff::Attributes::OnOff::Id) || (!dev->IsReachable()),
-                      Protocols::InteractionModel::Status::Failure);
+    VerifyOrReturnError(!((attributeId != OnOff::Attributes::OnOff::Id) || (!dev->IsReachable())),
```

Comment:
```suggestion
    VerifyOrReturnError((attributeId == OnOff::Attributes::OnOff::Id) && dev->IsReachable(),
```

Ovreall the adding `!` in front of compund bool makes the code harder to read throughout. This could have been split into two as well for each for the branches.